src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -124,9 +125,9 @@ const UsersFragment = ({
     };
 
     if (willFollow) {
-      await setFollow(data);
+      await setFollow(data, client);
     } else {
-      await setUnFollow(data);
+      await setUnFollow(data, client);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setFollow` and `setUnFollow` functions are now receiving an additional parameter `client`. Ensure that these functions have been updated to handle this new parameter. If they haven't been updated, this could lead to unexpected behavior or errors.
```diff
-      await setFollow(data, client);
-      await setUnFollow(data, client);
+      await setFollow(data);
+      await setUnFollow(data);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->